src/memory_server.ml

```diff
@@ -278,22 +298,45 @@ let calculate_boot_time_host_free_memory () =
       pages_to_kib (Int64.of_nativeint boot_time_host_free_pages) in
     Int64.mul 1024L boot_time_host_free_kib
 
+let calc_constant_boot_time_host_free_memory constant_count interval =
+  let tolerance = 10L in
+  debug "Check boot time host free memory: constant-count=%d check-interval=%f(seconds)" constant_count interval;
+  let rec calc_constant last same_count =
+    let free_memory = calculate_boot_time_host_free_memory () in
+    debug "Calculating boot time host free memory: %Ld" free_memory;
+    let same_count = match last with
+        | Some old_free_memory when Int64.abs(Int64.sub free_memory old_free_memory) <= tolerance -> same_count + 1
+        | _ -> 1
+    in
+    if same_count >= constant_count then begin
```

Comment:
`constant_count` - a better name would be : `same_count_min` (or similar) - the minimal number of counts we want to see before this is considered stable.

Reply:
an alternative:
```
let is_stable count = count >= constant_count in
[...]
if is_stable same_count then ...
```

Reply:
Yes `same_count_min` is a better name.